platform/android/export/export_plugin.cpp

```diff
@@ -856,9 +856,9 @@ bool EditorExportPlatformAndroid::_has_manage_external_storage_permission(const
 }
 
 bool EditorExportPlatformAndroid::_uses_vulkan() {
-	String current_renderer = GLOBAL_GET("rendering/renderer/rendering_method.mobile");
-	bool uses_vulkan = (current_renderer == "forward_plus" || current_renderer == "mobile") && GLOBAL_GET("rendering/rendering_device/driver.android") == "vulkan";
-	return uses_vulkan;
+	String rendering_method = GLOBAL_GET("rendering/renderer/rendering_method.mobile");
+	String rendering_driver = GLOBAL_GET("rendering/rendering_device/driver.android");
```

Comment:
This is still a bit unsafe btw, as there are other potential overrides that may conflict, e.g. a user could choose to set `rendering/renderer/rendering_method.android` or `rendering/rendering_device/driver.mobile`. It's not the default overrides we register but they can be added from the project settings. Not sure which one would take precedence.

They could also remove any `.mobile` and `.android` override and just have `rendering/rendering_device/driver` as the sole source of truth.

This isn't necessarily something that can be fixed here, I'm just pointing out how brittle our system is.

Reply:
On the other hand, one thing we could simplify here: Vulkan is the only possible driver for RenderingDevice-based rendering methods. Or conversely, it's only if it's using GL Compatibility that it's not going to use Vulkan.

So all this could be simplified to:
```
bool EditorExportPlatformAndroid::_uses_vulkan() {
	return GLOBAL_GET("rendering/renderer/rendering_method.mobile") != "gl_compatibility";
}
```
Still with the above caveat that there could be a `.android` override or no override if users added/removed them, so this could still break.

I assume the code was written as an explicit check of Vulkan being enabled instead of Vulkan *not* being enabled to future proof / not make assumptions on future available rendering methods and drivers, which is probably a good idea.

Reply:
I think I have old PR to resolve this kind of overrides on export.

Reply:
The non-standard override is relevant for any project setting, not just these.

Reply:
https://github.com/godotengine/godot/pull/71542 export code is obviously outdated (I'll update it for 4.5), but core part should be OK.